zeppelin-web/pom.xml

```diff
@@ -33,9 +33,9 @@
   <name>Zeppelin: web Application</name>
 
   <!-- See https://github.com/eirslett/frontend-maven-plugin/issues/229 -->
-  <prerequisites>
-    <maven>3.1.0</maven>
-  </prerequisites>
+  <!--<prerequisites>-->
+    <!--<maven>3.1.0</maven>-->
```

Comment:
Is there a reason to remove this one? 

Quick search\blame for this line points to [ZEPPELIN-622](https://github.com/apache/incubator-zeppelin/commit/48f875d42b90b6b175aa5deed2a552af8183e4a4) as a reason it was introduced in the first place